entries/admin.md

```diff
@@ -3,25 +3,25 @@ title: "General admin content"
 date: last-modified
 ---
 
-## Git tags and releases
-
-With the Seedcase product, Git tags and releases are/will be added
-fairly regularly. On the other hand, with the website, we only tag a
-version of the website after a major "milestone" or deadline. For
-instance, after sending the design documents to get reviewed or when
-sending an update document to our funder or stakeholders.
-
 ## GitHub repository settings
 
 -   Always
     [protect](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule)
     the main branch of the repository to limit forced pushes and
     deletions.
+- Branch protection settings to enable are:
+    - Require approval of at least 2
```

Comment:
at least 2 *developers* or *team members*

Reply:
I think 2 might be too much... but we can assess as we go along.